tests/e2e/testsuites/testsuites.go

```diff
@@ -334,7 +423,7 @@ func (t *TestPersistentVolumeClaim) Cleanup() {
 	// attempts may fail, as the volume is still attached to a node because
 	// kubelet is slowly cleaning up the previous pod, however it should succeed
 	// in a couple of minutes.
-	if t.persistentVolume.Spec.PersistentVolumeReclaimPolicy == v1.PersistentVolumeReclaimDelete {
+	if t.persistentVolume != nil && t.persistentVolume.Spec.PersistentVolumeReclaimPolicy == v1.PersistentVolumeReclaimDelete {
```

Comment:
Hmm curious, did you get a panic because of the missing check here?

Reply:
Yes, PV is nil in this case

Reply:
Hmm I hope we're not hiding a "should not happen" case here. I'll take another look later. Should be fine for now.